.drone.yml

```diff
@@ -204,7 +204,7 @@ steps:
           - master
           - feature/*
       event: pull_request
-
+  
```

Comment:
keep the blank line but remove the space at the beginning

Reply:
done